packages/generator/README.md

```diff
@@ -21,3 +27,130 @@ For installation and usage, see [Measurement Plug-In SDK for Python (`ni-measure
 - [click >= 8.1.3](https://pypi.org/project/click/8.1.3/)
 
 ---
+
+## Developing Measurements: Quick Start
+
+For installation and usage, see [Measurement Plug-In SDK Service for Python - Developing Measurements: Quick Start](https://pypi.org/project/ni_measurement_plugin_sdk_service/#developing-measurements-quick-start) section.
+
+---
+
+## Generating Measurement Clients: Quick Start
+
+This section provides instructions to generate custom measurement clients in Python using Measurement Plug-In SDK for Python.
+
+### Installation
+
+Make sure the system has the recommended Python version is installed. Install Measurement Plug-In SDK Service for Python using [pip](https://pip.pypa.io/).
+
+``` cmd
+REM Activate the required virtual environment if any.
+pip install ni-measurement-plugin-sdk-service
+```
+
+Check if you have installed the expected version of Measurement Plug-In SDK Service for Python installed by running the below command:
+
+```cmd
+pip show ni-measurement-plugin-sdk-service
+```
```

Comment:
Wrong package?

Reply:
Yes! I have changed the documentation to install only -sdk package.